js/src/dropdown.js

```diff
@@ -106,6 +106,7 @@ class Dropdown extends BaseComponent {
     this._inNavbar = this._detectNavbar()
 
     this._addEventListeners()
+    Data.setData(this._element, DATA_KEY, this)
```

Comment:
Please revert this; it shouldn't be needed anymore since we do this in the Base Component constructor

Reply:
@XhmikosR this was already there, you mean revert it to Data.setData(element, DATA_KEY, this) ?

Reply:
You added this line, it's not present on main.